recipes/default.rb

```diff
@@ -32,11 +32,32 @@
   mode 00755
 end
 
-execute 'install homebrew' do
-  command homebrew_go
-  environment lazy { { 'HOME' => ::Dir.home(homebrew_owner), 'USER' => homebrew_owner } }
-  user homebrew_owner
-  not_if { ::File.exist? '/usr/local/bin/brew' }
+begin
+  template '/etc/sudoers.d/homebrew_sudo' do
```

Comment:
`/etc/sudoers.d` can only be missing if someone messed around with the system, not sure we need to account for people's tinkering with system paths.